spec/displaynames.html

```diff
@@ -68,8 +151,13 @@ <h1>Intl.DisplayNames ( _locales_, _options_ )</h1>
         1. Let _dataLocaleData_ be _localeData_.[[&lt;_dataLocale_&gt;]].
         1. Let _types_ be _dataLocaleData_.[[types]].
         1. Assert: _types_ is a Record (see <emu-xref href="#sec-Intl.DisplayNames-internal-slots"></emu-xref>).
+        1. Let _languageDisplay_ be ? GetOption(_options_, *"languageDisplay"*, *"string"*, &laquo; *"dialect"*, *"standard"* &raquo;, *"dialect"*).
```

Comment:
Is it intentional to read `languageDisplay` even if `type` is not "language"? The consequences are observable to ECMAScript code, e.g.
```js
new Intl.DisplayNames(undefined, {
  type: "currency",
  get languageDisplay(){ throw new Error("[[Get]] languageDisplay") }
})
```

Reply:
Please see https://github.com/tc39/intl-displaynames-v2/issues/34
```
4. Intl.DisplayNames ( locales, options )
Normal convention should still be to always validate all options, even when unused.
That means the GetOption(options, "languageDisplay", ...) step should happen before If type is "language", then. (But don't move assigning displayNames.[[LanguageDisplay]] outside of the if-step.)
For example compare to SetNumberFormatUnitOptions().
```

Reply:
@anba